core/src/main/java/bio/terra/pearl/core/service/address/SmartyClientProvider.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.pearl.core.service.address;
+
+import com.smartystreets.api.ClientBuilder;
+import com.smartystreets.api.StaticCredentials;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SmartyClientProvider {
```

Comment:
I'm not sure we really need this to be a component?  I wonder if it should be a utility method "buildStaticCredentials()"

Reply:
I agree with you - I didn't originally code it that way. But doing it this way made it possible to make the tests. I couldn't figure out how to do mocking when I constructed the client inside of the constructor of the smart us client. I think the number of similarly-named clients within this package bothers me, but I haven't seen a clear way to make it less ambiguous.